tensorflow/lite/kernels/internal/optimized/optimized_ops.h

```diff
@@ -2337,14 +2337,12 @@ void BroadcastDivSlow(const ArithmeticParams& params,
 // BroadcastDiv is intentionally duplicated from reference_ops.h.
 // For more details see the comment above the generic version of
 // BroadcastDivSlow.
-template <int N = 5>
-inline void BroadcastDivSlow(const ArithmeticParams& params,
-                             const RuntimeShape& unextended_input1_shape,
-                             const uint8* input1_data,
-                             const RuntimeShape& unextended_input2_shape,
-                             const uint8* input2_data,
-                             const RuntimeShape& unextended_output_shape,
-                             uint8* output_data) {
+template <typename T, int N = 5>
+inline void BroadcastDivSlowQuantized(
+    const ArithmeticParams& params, const RuntimeShape& unextended_input1_shape,
+    const T* input1_data, const RuntimeShape& unextended_input2_shape,
+    const T* input2_data, const RuntimeShape& unextended_output_shape,
+    T* output_data) {
```

Comment:
All changes in this file have been copied from [here](https://github.com/tensorflow/tensorflow/blob/3093b7e22d0eb0068f35b64c9765bf7a96534752/tensorflow/lite/kernels/internal/reference/div.h#L102-L178) since this implementation is just a copy of the reference ops and these changes are required for `int8` support.

The comment seems to mention that this copy is intentional but I'm happy do remove the implementation if that would be preferred.
https://github.com/tensorflow/tensorflow/blob/3093b7e22d0eb0068f35b64c9765bf7a96534752/tensorflow/lite/kernels/internal/optimized/optimized_ops.h#L2287-L2292